### PR TITLE
Improve webapp provisioning to address #83

### LINKED ISSUE #83: Provision for Node should redeploy the webapp
Currently its a pain to have to destroy and rebuild the node instance each time the webapp is changed. The new webapp should be copied and restarted when provisioning the node instance.

### PR DESCRIPTION
**Pull request by risdenk**
Fixes #83 by removing the unnecessary install and removing the webapp folder each time before copying it.

### CODE REVIEW

File: provisioning/playbooks/nodejs/setup_elasticsearch-twitter-webapp.yml [hunk 1]
```diff
@@ -4,16 +4,16 @@
   gather_facts: False
   vars:
     nodejs_app_name: elasticsearch-twitter-webapp
-    nodejs_app_dir: /vagrant/{{ nodejs_app_name }}
     app_port: 8000
   tasks:
+    - name: remove app
+      command: rm -rf /opt/{{ nodejs_app_name }}
```
Comment (kruthar) [inline provisioning/playbooks/nodejs/setup_elasticsearch-twitter-webapp.yml]: Can be written with an Ansible files command

File: provisioning/playbooks/nodejs/setup_elasticsearch-twitter-webapp.yml [hunk 2]
```diff
@@ -4,16 +4,16 @@
...
+
     - name: copy app
-      command: cp -rf {{ nodejs_app_dir }} /opt/
-      args:
-        creates: /opt/{{ nodejs_app_name }}/
+      command: cp -rf /vagrant/{{ nodejs_app_name }} /opt/
```
Comment (kruthar) [inline provisioning/playbooks/nodejs/setup_elasticsearch-twitter-webapp.yml]: Can be written with an Ansible copy command

**Comment by risdenk** [comment]
@kruthar addressed the comments. tested and seems to work ok.

**Comment by kruthar** [comment]
:+1:

**Pull request merged**